profiles/utils.py

```diff
@@ -173,3 +245,67 @@ def update_full_name(user, name):
     if len(name_parts) > 1:
         user.last_name = ' '.join(name_parts[1:])[:30]
     user.save()
+
+
+def dict_to_style(style_dict):
+    """
+    Transform a dict into a string formatted as an HTML style
+
+    Args:
+        style_dict(dict): A dictionary of style attributes/values
+
+    Returns:
+        str: An HTML style string
+    """
+
+    return '; '.join(['{}: {}'.format(k, v) for k, v in style_dict.items()])
+
+
+def get_svg_avatar(text, size, color, bgcolor):
+    """
+    Generate an SVG avatar based om input text.  Adopted from https://github.com/CraveFood/avinit
+
+    Args:
+        text(str): The text to extract two initials from.
+        size(int): The width (and height) of the output SVG image
+        color(str): The font color code (minus the # prefix).
+        bgcolor(str): The image background color (minus the # prefix)
+
+    Returns:
+        str: an SVG image.
+    """
+
+    initials = '=)'
+
+    text = text.strip()
+    if text:
+        split_text = text.split(' ')
+        if len(split_text) > 1:
+            initials = split_text[0][0] + split_text[-1][0]
+        else:
+            initials = split_text[0][0]
```

Comment:
I think the function to derive initials from the text should be split out into a separate method. `initials = '=)'` should stay in this method and be called `default_text` or something